mlflow/models/cli.py

```diff
@@ -125,28 +125,20 @@ def serve(
 )
 @cli_args.ENV_MANAGER
 @cli_args.INSTALL_MLFLOW
-def predict(
-    model_uri,
-    input_path,
-    output_path,
-    content_type,
-    env_manager,
-    install_mlflow,
-):
+@click.option(
+    "--pip-requirements-override",
+    "-r",
+    default=None,
+    help="Specify packages and versions to override the dependencies defined"
+    "in the model. Must be a comma-separated string like x==y,z==a.",
+)
+def predict(**kwargs):
```

Comment:
curious why we need to use `kwargs`?

Reply:
To make it explicit that the Python API and CLI are same (and they should be)

Reply:
It looks like this is the simplest way. LGTM